aws/data_source_aws_instance_test.go

```diff
@@ -571,100 +589,158 @@ data "aws_instance" "foo" {
 }
 `
 
-const testAccInstanceDataSourceConfig_privateIP = `
-resource "aws_vpc" "foo" {
+func testAccInstanceDataSourceConfig_privateIP(rName string) string {
+	return fmt.Sprintf(`
+data "aws_availability_zones" "current" {
```

Comment:
Changed to avoid errors like:
>  Unsupported: Your requested instance type (t2.micro) is not supported in your requested Availability Zone (us-west-2d). Please retry your request by not specifying an Availability Zone or choosing us-west-2a, us-west-2b, us-west-2c.